src/Slacklinker/Sender/UserDataUpload/Types.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Slacklinker.Sender.UserDataUpload.Types where
+
+import Slacklinker.Prelude
+import Slacklinker.Types (Emoji)
+
+data UserData = UserData
+  { callsign :: Maybe Emoji
+  -- ^ Emoji callsign
+  , email :: Text
+  , gitHubUsername :: Maybe Text
+  }
+  deriving stock (Show)
```

Comment:
Does this have to be called `callsign`, or could we just call it `emoji`, so it's less Mercury-specific?

Reply:
Unfortunately that's the schema we have.

Reply:
The culture site's schema?

Reply:
Yeah, that's what the org chart response looks like :/ I could maybe manually write the deserialization so it accepts either one?

Reply:
Yeah I think that'd be good. Or call it `MercuryUserData` or something.